src/app/config/urls.ts

```diff
@@ -195,6 +195,17 @@ export const getLocationEntryPath = (location: Location, accession: string) =>
 export const getLocationEntryPathFor = (location: Location) =>
   partial(getLocationEntryPath, location);
 
+export const getUrlToBLAST = (
+  primaryAccession: string,
+  options?: {
+    start: number;
+    end: number;
+  }
+) =>
+  `${LocationToPath[Location.Blast]}?ids=${primaryAccession}${
+    options ? `[${options.start}-${options.end}]` : ''
+  }`;
+
```

Comment:
```suggestion
export const getURLToJobWithData = (
  jobType: JobTypes,
  primaryAccession: string,
  options?: {
    start: number;
    end: number;
  }
) =>
  `${jobTypeToPath(jobType)}?ids=${primaryAccession}${
    options ? `[${options.start}-${options.end}]` : ''
  }`;

```

Knowing that this is a common file I wouldn't make it specific. If it becomes too specific it should probably live somewhere in src/tools/blast and depend on `jobTypeToPath` as this is already a function to generate paths to a job form